src/Agent/NewRelic/Profiler/Profiler/SystemCalls.h

```diff
@@ -109,16 +109,18 @@ namespace NewRelic { namespace Profiler
 
                 if (!result)
                 {
+                    // log the error but don't throw
                     auto error = ::GetLastError();
-                    LogError(L"Unable to get the parent process path (1).  Error number: ", std::hex, error, std::resetiosflags(std::ios_base::basefield));
-                    throw ProfilerException();
+                    LogTrace(L"Unable to get the parent process path (1).  Error number: ", std::hex, error, std::resetiosflags(std::ios_base::basefield));
```

Comment:
I'm wondering if "warning" or "info" might be the appropriate level so that if any customers are impacted by this in the field, they'll see the message at the default agent logging level.

Reply:
I don't think that's necessary. We don't even know if the parent path is relevant until later in the decision process, and we already log the process and parent name as part of the w3wp check. If the parent name comes up blank, that should be a good enough clue.